src/editor/sidebar/index.js

```diff
@@ -209,7 +209,10 @@ class Sidebar extends Component {
 					<Button
 						isLink
 						onClick={ () => this.updateSender( senderName, senderEmail ) }
-						disabled={ inFlight || ( senderEmail.length ? ! hasValidEmail( senderEmail ) : false ) }
+						disabled={
+							inFlight ||
+							( senderEmail && senderEmail.length ? ! hasValidEmail( senderEmail ) : false )
+						}
```

Comment:
That won't be needed, #119 fixed it

Reply:
Removed in https://github.com/Automattic/newspack-newsletters/pull/123/commits/ce6d7676b4408dce9c376dc21ff61a3da2c5b5d9